src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_DRIVER =
+            "com.mysql.cj.jdbc.Driver";
+    private static final String DB_URL =
+            "jdbc:mysql://localhost:3306/jdbc";

```

Comment:
```suggestion
    private static final String DB_DRIVER = "com.mysql.cj.jdbc.Driver";
    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc";
```

also `jdbc` is pretty bad name for the database - it may cause you problems

I would recommend renaming it